five-minute-serverless-backend/README.md

```diff
@@ -0,0 +1,209 @@
+# Five Minute Serverless Backend
+*How to build a **serverless backend** for your web applciation and have it up in running on the cloud in minutes.*
+
+This tutorial takes you through 5 steps to building a backend for your web application:
+1. Setup your Binaris environment
+1. Create a serverless function for your backend code
+1. Update your function to work from a browser
+1. Call your function from frontend code
+
+## Setup your Binaris environment
+
+If you do not have a Binaris account, [sign up here](https://binaris.com/signup?t=XXXXXX). Binaris is free to use up to 400,000 GB-seconds a month. At the end of the sign-up process you will receive an account id and API key. Save them for the next steps.
+
+Install the CLI:
+
+```
+$ npm install binaris -g
+```
+
+Run `bn login` and paste in the API key received above to authenticate your account.
+
+## Create a serverless function for your backend code
+
+Create a new folder and use the CLI to create the template code for your function:
+
+```
+$ mkdir backend
+$ cd backend
+$ bn create node8 public_backend --executionModel concurrent
+Created function public_backend in .../backend
```

Comment:
why three dots? use a real dir please (maybe `~`?)
(i feel like a parrot repeating stuff. this is the third time i'm commenting about this without a response)

Reply:
It could be anywhere. Not fixed

Reply:
so `/.../backend`? afaict `bn` print full path.